tools/plugins/com.liferay.ide.project.core/src/com/liferay/ide/project/core/PluginsSDKProjectProvider.java

```diff
@@ -339,7 +342,19 @@ public ILiferayProject provide( Object type )
 
             try
             {
-                if ( SDKUtil.isSDKProject( project ) )
+                boolean hasNewSdk = false;
```

Comment:
Add this check as a static method on the SDKClasspathContainer method, it should now how to check for itself.